docs/src/components/ComponentPage/AndroidComponentPage.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow strict */
+
+import React from 'react';
+import BpkLink from 'bpk-component-link';
+
+import ComponentScreenshots from '../DocsPageBuilder/ComponentScreenshots';
+
+import ComponentPage from './ComponentPage';
+
+type Props = {
+  documentationId: string,
+  readme: string,
+  screenshots: [
+    {
+      id: string,
+      screenshots: {
```

Comment:
Why is this `screenshots`(plural) when the value is an object(singular)

Reply:
The Flow type was wrong here, I should have made it an array. I expect it wasn't caught because the file that consumes this doesn't have the Flow annotation. I'm fixing that now (and for the other files I created/modified)

Reply:
done